EmarsysInbox/EmarsysInbox.storyboard

```diff
@@ -82,7 +82,7 @@
                                                         <constraint firstItem="6Jn-no-g7Z" firstAttribute="leading" secondItem="MUT-IZ-idZ" secondAttribute="leading" id="l2J-mv-ds1"/>
                                                     </constraints>
                                                 </view>
-                                                <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleAspectFit" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo" translatesAutoresizingMaskIntoConstraints="NO" id="kyd-LL-fHE">
+                                                <imageView clipsSubviews="YES" userInteractionEnabled="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" verticalHuggingPriority="251" image="logo" translatesAutoresizingMaskIntoConstraints="NO" id="kyd-LL-fHE">
```

Comment:
we may want to keep it `scaleAspectFit`, otherwise image may be 'stretched' unexpectedly

Reply:
or `scaleAspectFill`